recipes/gtabview/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "gtabview" %}
+{% set version = "0.10" %}
+{% set sha256 = "f791b0a103c9eeb9aa7c76c47c85f16d682d77c07a6f433a0d8a50bd9e7df7e1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} setup.py install"
+  entry_points:
+    - gtabview = gtabview_cli.gtabview:main
+
+requirements:
+  host:
+    - python
+    - setuptools
+  run:
+    - python >=2.7
```

Comment:
Use the same min python on host and run.

Reply:
Done